Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,20 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Decode Encodable object from file.
+    ///
+    /// - Parameters:
+    ///   - url: URL a file with saved object data.
+    ///   - type: type of the object that is being decoded.
+    ///   - decoder: JSONDecoder.
+    /// - Throws: Throws any errors thrown by JSONDecoder.
+    /// - Returns: An object of specified type. Returns nil if failed to read from URL.
+    public func decode<T: Decodable>(from url: URL, as type: T.Type, using decoder: JSONDecoder = JSONDecoder()) throws -> T? {
```

Comment:
I think that since your `encode` function's parameters are ordered as
`encode(_ object: T, to url: URL, using encoder: JSONEncoder...)`
then it would make sense for this to be
`decode(_ type: T.Type, from url: URL, using decoder: JSONDecoder)`

Reply:
The order of arguments now matched to `encode` method